ui/app/registration/services/patientService.js

```diff
@@ -29,6 +29,23 @@ angular.module('bahmni.registration')
             };
             return patientServiceStrategy.search(config);
         };
+        var searchDuplicatePatients = function (query, gender, birthDate, contactNumber) {
+            var config = {
```

Comment:
@IipperMangesh the following fields are missing on the duplicate check: System ID, National ID and ART Number

Reply:
@IipperMangesh Sorry please ignore the above comment; the duplicate check must be based on first name, last name, DOB, Gender and exact phone number.

src: ticket description
![image](https://user-images.githubusercontent.com/22587098/53948099-c213db80-40cf-11e9-9a24-d61ade88b33c.png)